src/components/dropdowns/dropdown/DropdownCSS.js

```diff
@@ -0,0 +1,34 @@
+import { colors } from "../../colors/index";
+
+export const dropdownWrapperCSS = `
+	font-family: "HK Grotesk", sans-serif;
+	border-radius: 5px;
+	border-style: solid;
+	border-width: 1px;
```

Comment:
the border is not very visible when the dropdown isn't open. i say we go with 2px